src/hotspot/share/prims/vectorSupport.cpp

```diff
@@ -88,20 +88,13 @@ jint VectorSupport::klass2length(InstanceKlass* ik) {
 
 void VectorSupport::init_payload_element(typeArrayOop arr, bool is_mask, BasicType elem_bt, int index, address addr) {
   if (is_mask) {
-    // Masks require special handling: when boxed they are packed and stored in boolean
```

Comment:
The comment is still valid. Please, keep it and update with latest details about mask support.